### PR TITLE
Ensure learning materials and search suggestions are specified for every enabled layer

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/app/utils/route-utils.ts [hunk 1]
```diff
@@ -6,14 +6,23 @@ import { EvmTokenType, Layer } from '../../oasis-indexer/api'
...
+export type SpecifiedPerEnabledLayer<T = any> = {
+  [N in keyof (typeof RouteUtils)['ENABLED_LAYERS_FOR_NETWORK']]: {
+    [L in keyof (typeof RouteUtils)['ENABLED_LAYERS_FOR_NETWORK'][N]]: T
   }
+}
+
+export abstract class RouteUtils {
+  private static ENABLED_LAYERS_FOR_NETWORK = {
+    [Network.mainnet]: {
+      [Layer.emerald]: true,
```
Comment (csillag) [inline src/app/utils/route-utils.ts:18]: Technically, this should be disabled for now, too. (So commented out.)
(It was an oversight that I have accidentally re-enabled it at one point, by leaving in a debug change while merging.)

Comment (lukaw3d) [reply src/app/utils/route-utils.ts:18]: Okay ill try to find a sapphire verified contract example first

Comment (lukaw3d) [reply src/app/utils/route-utils.ts:18]: I have not found it :shrug:

**Review by csillag — APPROVED** [review]
Reviewed, LGTM.

(There or some things that we need to discuss, but not strictly related to this PR, so whatever.)

**Pull request merged**